rules/misc.yaml

```diff
@@ -105,3 +105,34 @@ CAMERA:
 		Image: camera
 	EditorOnlyTooltip:
 		Name: (reveals area to owner)
+
+FLAMEGUY:
+	Inherits@1: ^ExistsInWorld
+	Inherits@2: ^SpriteActor
+	Mobile:
+		Speed: 71
+		SharesCell: true
+		TerrainSpeeds:
+			Clear: 45
+			Road: 50
+			Rail: 45
+			DirtRoad: 50
+			Rough: 40
+			Ore: 45
+			Gems: 45
+	HiddenUnderFog:
+	RenderSprites:
+		Palette: effect
+	WithInfantryBody:
+		IdleSequences: run
+	Health:
+		HP: 160
+	SelfHealing:
+		Step: -10
+		HealIfBelow: 101
+	ScaredyCat:
+	WithDeathAnimation:
+		FallbackSequence: die
+		UseDeathTypeSuffix: false
+		DeathSequencePalette: effect
+		DeathPaletteIsPlayerPalette: False
```

Comment:
Minor nit: Please use a lowercase `false` as we/you did in the rest of the file.